manager/src/test/java/com/redhat/service/smartevents/manager/workers/resources/BridgeWorkerTest.java

```diff
@@ -115,6 +119,11 @@ void testProvisionWorkWithKnownResource(ManagedResourceStatus status,
         assertThat(refreshed.getDependencyStatus()).isEqualTo(dependencyStatusWhenComplete);
         verify(rhoasServiceMock).createTopicAndGrantAccessFor(TEST_TOPIC_NAME, RhoasTopicAccessType.CONSUMER_AND_PRODUCER);
         verify(workManager, times(isWorkComplete ? 0 : 1)).reschedule(any());
+
+        if (throwRhoasError) {
```

Comment:
Woot! A single test showing records are created if the `Worker` fails..

If my approach seems reasonable I'd obviously write a lot more tests...